### PR TITLE
Enable result sets in webviews

### PR DESCRIPTION
**Pull request by worksofliam**
Adds a new button the execute dropdown to run the statement in a new view.

This will run the statement in a brand new tab instead of the standard webview.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -314,6 +314,12 @@
         "mac": "cmd+r",
         "when": "editorLangId == sql && resourceExtname != .inb"
       },
+      {
+        "command": "vscode-db2i.runEditorStatement.inView",
+        "key": "ctrl+alt+r",
+        "mac": "cmd+ctrl+r",
+        "when": "editorLangId == sql"
```
Comment (davecharron) [inline package.json:321]: Should this have the same resource extension check as at line 315?

**Review by davecharron — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (worksofliam) [reply package.json:321]: I didn't think so. It might be nice to take a statement from a Notebook and run it in a new Window

**Comment by forstie** [comment]
Two comments:
1) For longer running SQL statements, I could not see the SQL cancel button when using the enhancement
2) It would be nice to give users a method to control the name of the tab/window that is being opened

**Review by forstie — APPROVED** [review]
I'm seeing the expected behavior now. Thanks

**Pull request merged**